Dockerfile

```
@@ -17,12 +17,14 @@ RUN export GOOS=$TARGETOS && \
     export GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3 | tr -d 'v') && \
     make build
 
-FROM k8s.gcr.io/build-image/debian-iptables:bullseye-v1.1.0 AS nmi
+FROM k8s.gcr.io/build-image/debian-iptables:bullseye-v1.2.0 AS nmi
 # upgrading libssl1.1 due to CVE-2021-3711 and CVE-2021-3712
 # upgrading libgssapi-krb5-2 and libk5crypto3 due to CVE-2021-37750
 # upgrading libgmp10 due to CVE-2021-43618
 # upgrading bsdutils due to CVE-2021-3995 and CVE-2021-3996
-RUN clean-install ca-certificates libssl1.1 libgssapi-krb5-2 libk5crypto3 libgmp10 bsdutils
+# upgrading libc6 due to CVE-2022-23218
+# upgrading libc-bin due to CVE-2022-23218
+RUN clean-install ca-certificates libssl1.1 libgssapi-krb5-2 libk5crypto3 libgmp10 bsdutils libc6 libc-bin
```

We can remove `libgssapi-krb5-2` and `libk5crypto3` from this `clean-install` statement since they are upgraded in `bullseye-v1.2.0`. Let's also include `libsystemd0` and `libudev1` (exact same change in https://github.com/Azure/azure-workload-identity/pull/404/files)

```suggestion
RUN clean-install ca-certificates libssl1.1 libgssapi-krb5-2 libk5crypt libgmp10 bsdutils libsystemd0 libudev1
```